sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -86,9 +86,10 @@ SLF4J is only needed if you plan to use logging, please also download an SLF4J b
 ### Setup Configuration Class
 - In order to set up configuration class, you'll need to extend `AbstractCosmosConfiguration`
 
-- Azure-spring-data-cosmos also supports `Response Diagnostics String` and `Query Metrics`.
+- Azure-spring-data-cosmos also supports `Response Diagnostics String`, `Query Metrics` and `Max Degree of Parallelism`.
 Set `queryMetricsEnabled` flag to true in application.properties to enable query metrics.
 In addition to setting the flag, implement `ResponseDiagnosticsProcessor` to log diagnostics information.
+Set `maxDegreeOfParallelism` flag to an integer in application.properties to allow parallel processing.
```

Comment:
We should also mention that setting the value to -1 will lead the SDK to decide an optimal value.

Reply:
Done.